src/super_gradients/training/utils/callbacks.py

```diff
@@ -237,7 +237,25 @@ def __init__(self, phase, initial_lr, update_param_groups, train_loader_len, net
         self.net = net
         self.training_params = training_params
 
-    def __call__(self, *args, **kwargs):
+    def __call__(self, context: PhaseContext, **kwargs):
+        if self.enable_lr_scheduling(context):
+            self.perform_scheduling(context)
+
+    def enable_lr_scheduling(self, context: PhaseContext):
```

Comment:
the naming sounds like an action (like you enable something)
maybe change it to something like `is_lr_scheduling_enabled`

Reply:
done.